OasisDEX/oasis-borrow
# Added config parameters, remote top banner config

Top banner is now a part of the remote config.

## Changes 👷‍♀️
- new `getAppConfig` part - `parameters` - used for variables/strings which are prone to change
- used the new parameters in top banner
  
## How to test 🧪
- `oazo-configuration` now has a local server, just spin it up, change the env config url to point to this local server and change the values there, should be reflected

or

- change the values in `ob-config-overrides`, this will work as well
